src/Animated.test.js

```diff
@@ -0,0 +1,102 @@
+import { getNumberOfInitializedNodes_forTestsOnly as numberOfNodes } from './core/AnimatedNode';
```

Comment:
Looks like this isn't really necessary. I think what we could do instead is to have our `ReanimatedModule` mock to verify we have equal number of calls to `createNode` and `dropNode`